UPGRADE.md

```diff
@@ -10,7 +10,10 @@ Follow instructions in a [monorepo upgrade guide](./docs/contributing/UPGRADE.md
 
 ### You are developing a project based on project-base
 * upgrade only your composer dependencies and follow instructions
-* if you want update your project with the changes from [shopsys/project-base], you can follow the *(optional)* instructions or cherry-pick from the repository whatever is relevant for you but we do not recommend rebasing or merging everything because the changes might not be compatible with your project as it probably evolves in time
+* if you want update your project with the changes from [shopsys/project-base],
+    you can follow the *(optional)* instructions or cherry-pick from the repository whatever is relevant for you but we do not recommend rebasing or merging everything because the changes might not be compatible with your project as it probably evolves in time
+* instructions marked as *(optional)* are not vital, but could be helpful,
```

Comment:
this should be in a separate commit :slightly_smiling_face: it does not fit to "upgrade documentation: Removed ordered list of repositories maintained by monorepo"